modules/auxiliary/scanner/http/epmp1000_cmd_exec.rb

```diff
@@ -8,6 +8,9 @@ class MetasploitModule < Msf::Auxiliary
   include Msf::Auxiliary::AuthBrute
   include Msf::Auxiliary::Report
   include Msf::Auxiliary::Scanner
+  include Msf::Module::Deprecated
+
+  deprecated(Date.new(2017, 12, 29), 'auxiliary/scanner/admin/http/epmp1000_ping_cmd_exec')
```

Comment:
This is supposed to be the date the module will become deprecated.  This means it will be deprecated next week.  At least in experience with deprecation, we generally try to give a months notice.

Reply:
Yeah, 60 days for sure. Give people time to update their RC scripts and automated workflows and whatnot.